tests/Doctrine/Tests/Common/Annotations/AbstractReaderTest.php

```diff
@@ -20,90 +20,90 @@ public function testAnnotations()
         $class = $this->getReflectionClass();
         $reader = $this->getReader();
 
-        $this->assertCount(1, $reader->getClassAnnotations($class));
-        $this->assertInstanceOf($annotName = DummyAnnotation::class, $annot = $reader->getClassAnnotation($class, $annotName));
-        $this->assertEquals('hello', $annot->dummyValue);
+        self::assertCount(1, $reader->getClassAnnotations($class));
```

Comment:
Yes it is (as far as I know) that why I want to know why you @lcobucci preferer this version.